### ISSUE TITLE
Inner Classes Attribute Causes Compilation Errors

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/jeo-maven-plugin#589]
Despite that bytecode attributes are optional for bytecode execution, the absence of `InnerClasses` attributes causes compilation errors. I run `spring-fat` integration test in `opeo-manen-plugin` and faced with the following exception:

```java
[INFO] --- compiler:3.10.1:testCompile (default-cli) @ opeo-spring-fat-it ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /Users/lombrozo/Workspace/EOlang/opeo-maven-plugin/target/it/spring-fat/target/test-classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/lombrozo/Workspace/EOlang/opeo-maven-plugin/target/it/spring-fat/src/test/java/org/eolang/jeo/spring/FactorialApplicationTest.java:[39,48] cannot find symbol
  symbol:   variable WebEnvironment
  location: @interface org.springframework.boot.test.context.SpringBootTest
[ERROR] /Users/lombrozo/Workspace/EOlang/opeo-maven-plugin/target/it/spring-fat/src/test/java/org/eolang/jeo/spring/FactorialApplicationTest.java:[39,63] an enum annotation value must be an enum constant
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

The content of `FactorialApplicationTest.java` is following:

```java
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
class FactorialApplicationTest {
// some code here
}
```

As you can see it uses `SpringBootTest.WebEnvironment.RANDOM_PORT ` class. Actually it is the `SpringBootTest` class with the inner enum `WebEnvironment`. If I replace `SpringBootTest` which we disassembled/assembled with the original one (from jar), the compilation works correctly without any errors.

So I apply bytecode representation of failed `SpringBootTest` class and an original one:
[failed.txt](https://github.com/objectionary/jeo-maven-plugin/files/15296634/failed.txt)
[working.txt](https://github.com/objectionary/jeo-maven-plugin/files/15296638/working.txt)

As you can see, the main difference in `InnerClasses` attributes. So most probably it is a cause of the bug.

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#591](https://github.com/objectionary/jeo-maven-plugin/issues/591), [#592](https://github.com/objectionary/jeo-maven-plugin/issues/592), [#593](https://github.com/objectionary/jeo-maven-plugin/issues/593), [#594](https://github.com/objectionary/jeo-maven-plugin/issues/594), [#595](https://github.com/objectionary/jeo-maven-plugin/issues/595) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 4 puzzles [#591](https://github.com/objectionary/jeo-maven-plugin/issues/591), [#592](https://github.com/objectionary/jeo-maven-plugin/issues/592), [#593](https://github.com/objectionary/jeo-maven-plugin/issues/593), [#594](https://github.com/objectionary/jeo-maven-plugin/issues/594) are still not solved; solved: [#595](https://github.com/objectionary/jeo-maven-plugin/issues/595).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#593](https://github.com/objectionary/jeo-maven-plugin/issues/593), [#594](https://github.com/objectionary/jeo-maven-plugin/issues/594) are still not solved; solved: [#591](https://github.com/objectionary/jeo-maven-plugin/issues/591), [#592](https://github.com/objectionary/jeo-maven-plugin/issues/592), [#595](https://github.com/objectionary/jeo-maven-plugin/issues/595).

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#594](https://github.com/objectionary/jeo-maven-plugin/issues/594) is still not solved; solved: [#591](https://github.com/objectionary/jeo-maven-plugin/issues/591), [#592](https://github.com/objectionary/jeo-maven-plugin/issues/592), [#593](https://github.com/objectionary/jeo-maven-plugin/issues/593), [#595](https://github.com/objectionary/jeo-maven-plugin/issues/595).
